nuitka/build/SingleExe.scons

```diff
@@ -1596,7 +1596,7 @@ def discoverSourceFiles():
         return target_file
 
     # Scan for Nuitka created source files, and add them too.
-    for filename in os.listdir(source_dir):
+    for filename in sorted(os.listdir(source_dir)):
```

Comment:
Good, but more as a note to self, the scons file should start to use `nuitka.utils.FileOperations` in the mid run, which does this already.

Reply:
Also a note to this self, as I'll likely be poking around in here again.